contracts/starknet/lib/stark_eip191.cairo

```diff
@@ -94,12 +104,12 @@ namespace StarkSig:
 
         assert message[0] = STARKNET_MESSAGE
         assert message[1] = DOMAIN_HASH
-        assert message[2] = proposer_address
+        assert message[2] = authenticator
```

Comment:
why the switch to authenticator rather than proposer?

Reply:
I think the eip191 standard sets the data out this way: https://eips.ethereum.org/EIPS/eip-191.

Reply:
Ok yup sounds good :)